extensions/oidc-client/deployment/src/main/java/io/quarkus/oidc/client/deployment/OidcClientBuildStep.java

```diff
@@ -184,6 +191,27 @@ private AccessTokenInstanceBuildItem build() {
         return index.getIndex().getAnnotations(ACCESS_TOKEN).stream().map(ItemBuilder::new).map(ItemBuilder::build).toList();
     }
 
+    @BuildStep(onlyIfNot = IsNormal.class, onlyIf = GlobalDevServicesConfig.Enabled.class)
```

Comment:
Proposing to move it to a dedicated `OidcClientDevServiceBuildStep`, may be we can later share it with KeycloakAdminClient, etc

Reply:
@michalvavrik never mind, let's keep it here for now, I can check later if sharing some of it makes sense